src/components/sidepanel/ControlBar.js

```diff
@@ -34,8 +34,10 @@ class ControlBar extends Component {
   onClickReset = () => {
     this.props.Filters.reset();
   };
+
   onClickUpdate = () => this.props.Update.update();
-  onToggleLive = () => this.props.Time2.toggleLive();
+
+  onToggleLive = () => this.props.Time.toggleLive();
```

Comment:
This was probably just an issue where the wrong window was focused and I typed in a "2" without noticing that anything had gone wrong.